giskard/ml_worker/ml_worker.py

```diff
@@ -162,3 +166,4 @@ def stop(self):
         if self.ws_conn:
             self.ws_stopping = True
             self.ws_conn.disconnect()
+        shutdown_pool()
```

Comment:
should the shutdown be in the `finally` block?

Reply:
Not sure I understand this ? do you mean in the self.ws_conn ?

Reply:
never mind, I got confused by the GH code block being collapsed during the review, I thought this was part of another method